sdk/guides/04-customize-rewards.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Customize Rewards Logic
+slug: /guides/customize-rewards
+sidebar_label: Customize Rewards
+hide_table_of_contents: false
+---
+
+<intro-end />
+
+In this guide, we will walk through two different methods of customizing rewards logic within your metagraph. 
+
+
+## Understanding Rewards
+Rewards are emitted on every timed snapshot of the metagraph and increase the circulating supply of the metagraph token beyond the initial balances defined in genesis.csv. These special transaction types can be used to distribute your currency to fund node operators, or create fixed pools of tokens over time. 
+
+By default, no rewards are distributed by a metagraph using the Currency Framework which results in a static circulating supply. The rewards customizations described below create inflationary currencies - the rate of which can be controlled by the specific logic introduced. Similarly, a maximum token supply can easily be introduced if desired to prevent unlimited inflation.
```

Comment:
I would add a section here describing the context information available and describing the limits of customization here to 'sell' the benefits of this. 

I.e. the information you have available in this rewards function is all data produced by the prior incremental update, including any data produced. Additionally the reward function can use customized code that can invoke any library function of your choice.

These two things are important to mention (although they're partially demonstrated in examples.)